### PR TITLE
fix: use slash for filepath (#90), closes #67

### LINKED ISSUE #67: Renaming/deleting markdown file breaks reloading
#### Version

0.4.3

#### Steps to reproduce

1. run dev server
2. rename or delete a markdown file

#### What is expected?

Should handle renaming or deletion like Hexo, without breaking the workflow.

#### What is actually happening?

```
ERROR in <omitted>/pages/drafts/break.md?saberPage=61d9a790
Module build failed (from <omitted>/node_modules/saber/vue-renderer/lib/saber-page-loader.js):
Error: ENOENT: no such file or directory, open '<omitted>\pages\drafts\break.md'
...
```

### PR DESCRIPTION
**Pull request by MarchWorks**
**Summary**
Fixes: #67 
**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI/CSS related code, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: packages/saber/lib/plugins/source-pages.js [hunk 1]
```diff
@@ -112,7 +112,7 @@ exports.apply = api => {
         if (type === 'remove') {
           await api.hooks.manipulatePage.promise({
             action: 'remove',
-            id: hash(filepath)
+            id: hash(slash(filepath))
```
Comment (egoist) [inline packages/saber/lib/plugins/source-pages.js]: I think we don't need to slash-ify it here? We hash it anyways.

Comment (egoist) [reply packages/saber/lib/plugins/source-pages.js]: oh, now I see it.

**Comment by egoist** [comment]
You can claim your bounty at https://issuehunt.io/r/egoist/saber/issues/67, I know you don't go for money but it's better than nothing XD

btw, it seems I will get 20% too 🤔

**Comment by MarchWorks** [comment]
@egoist OK

**Comment by h404bi** [comment]
Nice catch!

**Pull request merged**